x-pack/plugins/apm/dev_docs/overflow_bucket_setup.md

```diff
@@ -1,11 +1,112 @@
+# Table of contents
+1. [Summary](#summary)
+2. [Pre-requisites](#prerequisites)
+3. [Using tilt](#tilt)
+4. [Other ways](#other)
+
 ## Summary
 
 8.7 introduced Overflow Buckets for Metrics. The below setup would enable generating the overflow buckets for the metrics indices.
 
-### Pre-requisites
+## Pre-requisites
 
 - Install golang and set go path properly
 
+## Using tilt
+
+With a 1GB server, MaxTransactionGroups is 5000 and MaxServices is 1000. Per-service max transaction groups is hardcoded as 10% of max transaction groups, i.e. 500.
+
+### Pre-requisites
+1. Clone [Apm-server](https://github.com/elastic/apm-server/tree/main) repository
+2. Install [Docker](https://www.docker.com/products/docker-desktop/).
+3. Install [Tilt](https://docs.tilt.dev/install.html).
+4. Execute `tilt up` in the root of the repository.
+
+For more detailed instructions you can check [apm-server dev documentation](https://github.com/elastic/apm-server/blob/main/dev_docs/TESTING.md#tilt--kubernetes).
```

Comment:
Can you add a note about what is `tilt up` is doing here? Is it only starting APM Server or also ES and Kibana?